docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -1,116 +1,91 @@
+However, when working on systems like the [interactive Yens](/_user_guide/yen){target="_blank"} where resources like **CPU**, **RAM**, and **disk space** are shared among many researchers, it is important that all users should be mindful of how their work impacts the larger community.
 
 !!! tip
-    When using interactive yens, use the ```htop``` and ```userload``` commands to monitor CPU and RAM usage. Use the ```gsbquota``` command to monitor disk quota.
+    When using interactive Yens, use the `htop` and `userload` commands to monitor CPU and RAM usage. Use the `gsbquota` command to monitor disk quota.
 
 ### CPU & RAM
+Per our [Community Guidelines](/_policies/user_limits){target="_blank"}, CPU usage should always be limited to 48 CPU cores/threads per user at any one time on Yen[2-5] and up to 12 CPU cores on Yen1. Some software (R and RStudio, for example) defaults to claiming all available cores unless told otherwise.
 
-Per our <a href="/yen/community.html" target="_blank">Community Guidelines</a>, CPU usage should always be limited to 48 CPU cores/threads per user at any one time on yen[2-5] and up to 12 CPU cores on yen1.
-Some software (R and RStudio, for example) default to claiming all available cores unless told to do otherwise.
-These defaults should always be overwritten when running R code on the yens. Similarly, when working with multiprocessing code in languages like Python,
-care must be taken to ensure your code does not grab everything it sees. Please refer to our parallel processing <a href="/topicGuides/index.html" target="_blank">Topic Guides</a> for information about how to limit resource consumption when using common packages.
+These defaults should always be overwritten when running R code on the Yens. Similarly, when working with multiprocessing code in languages like Python, care must be taken to ensure your code does not grab everything it sees. Please refer to our [Python parallel processing guidelines](/_user_guide/best_practices_parallel_processing_python){target="_blank"} for information about how to limit resource consumption when using common packages.
 
-One easy method of getting a quick snapshot of your CPU and memory usage is via the ```htop``` command line tool. Running ```htop``` shows usage graphs and a process list that is sortable by user, top CPU, top RAM, and other metrics. Please use this tool liberally to monitor your resource usage, especially if you are running multiprocessing code on shared systems for the first time.
+One easy method of getting a quick snapshot of your CPU and memory usage is via the `htop` command line tool. Running `htop` shows usage graphs and a process list that is sortable by user, top CPU, top RAM, and other metrics. Please use this tool liberally to monitor your resource usage, especially if you are running multiprocessing code on shared systems for the first time.
 
-The ```htop``` console looks like this:
+The `htop` console looks like this:
```

Comment:
We should add a new image here, that is no longer representative because the cpu bars are disabled by default (i think)
<img width="1590" alt="image" src="https://github.com/user-attachments/assets/6808dab7-f37c-4199-b0cd-cf1a1022dd36">